tests/reimplemented_interface.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Check if hooks are invoked only once for reimplemented interfaces
+--EXTENSIONS--
+otel_instrumentation
+--FILE--
+<?php
+interface A {
+    function m(): void;
+}
+interface B extends A {
+}
+class C implements A, B {
```

Comment:
Interfaces are not deduplicated, this class hierarchy adds the hook for `A::m()` twice, once for `C->A`, once for `C->B->A`.
https://github.com/open-telemetry/opentelemetry-php-instrumentation/blob/f2c1a1e87ef540f2d70e2d3bdc990a32f04307ec/otel_observer.c#L322-L330